package-lock.json

```diff
@@ -34680,6 +34650,16 @@
         "punycode": "^2.1.0"
       }
     },
+    "trezor-connect": {
```

Comment:
not sure if this change is expected since i don't see any dependency change on `package.json`